### DISCUSSION TITLE
SQL logging during seeding

### DISCUSSION THREAD

**Discussion by felipesere** [loco-rs/loco#959] [Q&A]
Not sure if I'm holding it wrong 🤷‍♂️

I'm trying to move my PostsgreSQL backed app to use Sqlite (for "reasons") and my current hurdle is that I get the following error when seeding the users:
```
called `Result::unwrap()` on an `Err` value: DB(Json("premature end of input"))
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Is there a way to turn on SQL logging for seeding? I have it turned on in my test config but I see nothing.
It's hard to tell which column gives me grief :)

**Comment by felipesere** [comment]
So I found my migration issue: the date times needed an extra `Z` at the end to be `DateTime<Utc>` but the question of logging would still be useful...

**Comment by jondot** [comment]
converted to issue, #1011 we'll take a deeper look
